### DISCUSSION TITLE
Sensor node restart

### DISCUSSION THREAD

**Discussion by bnurrr** [Security-Onion-Solutions/securityonion#7915] [Unsupported Versions] [answered]
Hello,

We had a Security Onion 2.3.120 distributed deployment with a single Manager node, a search node, a sensor node and a fleet node

How to safely restart sensor node  

Thanks,

**Comment by n0th1nt0lOOz** [comment]
you mean restart OS? just restart

**Comment by bnurrr** [comment]
no, I want to learn before sensor node restart, what am I need to do components (docker and containers) / restart or stop 
if i need to stop components, how to stop all components

**Comment by n0th1nt0lOOz** [comment, marked as answer]
you dont need to do anything...just restart
